ansible/roles/influxdb/defaults/main.yml

```diff
@@ -15,5 +15,5 @@
 # limitations under the License.
 #
 
-influxdb_rpm: influxdb-0.9.4.2-1.x86_64.rpm
-influxdb_checksum: "sha256:9a2b6cfdf70e0050a46d07cc0f4135a7538eaac1a3c1745d860d454179f771fb"
+influxdb_rpm: influxdb-1.8.3.x86_64.rpm
+influxdb_checksum: "sha256:74c606e2da6d33b193bc6224afb552b09567c7e1a31fee4e51ff48550f59a7cc"
```

Comment:
Do they have SHA-512 hashes published? If so, we should use those.

Reply:
I checked just now and they do have SHA-512. I'll run a quick test with the new hash and update the PR. Thanks!

Reply:
done